### DISCUSSION TITLE
existing placement_io file has basic uniform distribution of I/O pins within the core area. Does OpenROAD support this scenario?

### DISCUSSION THREAD

**Discussion by rr925** [The-OpenROAD-Project/OpenROAD#6105] [Q&A]
The GUI displays the I/O interfaces as shown：
<img width="568" alt="微信图片_20241106150353" src="https://github.com/user-attachments/assets/9afe00c8-a8ae-4428-827c-cbb90be48850">
When reading this I/O layout, an error occurred: “Pin location error.” I would like to know if OpenROAD supports only I/O interface distributions on the periphery of the layout.

**Comment by maliberty** [comment]
I don't understand how you can see it in the GUI and also have an error.  Can you provide a test case in an issue?

**Reply by rr925** [reply]
I’m glad to receive your response! .def file only gives the IO floorplan position, I converted it to an odb file and placed it in the GUI-generated IO layout, but when I proceeded to the next step, which was macro placement, errors occurred:Pin location error .
[case.zip](https://github.com/user-attachments/files/17644099/case.zip)

**Reply by rr925** [reply]
Do I need to modify the source code to achieve this?
